moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template

```diff
@@ -20,6 +20,9 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>xacro</run_depend>
+  
+  <!-- Commenting the warehouse_ros_mongo dependency until we have a clean release -->
```

Comment:
Sorry for nit-pick, but I would really prefer an explanation such as

> This package is referenced in the warehouse launch files, but does not build out of the box at the moment. Commented the dependency until this works.

"until we have a clean release" doesn't really give away the details...